### DISCUSSION TITLE
Airport gate - should it go into Places overlay or Things overlay?

### DISCUSSION THREAD

**Discussion by matkoniecz** [streetcomplete/StreetComplete#5892] [Ideas]
https://wiki.openstreetmap.org/wiki/Tag:aeroway%3Dgate - I think it would be worth adding them to overlays, after all these can be also not yet mapped or outdated. And it can be useful for orientation while mapping other features.

Looking at description it looks like Things overlay would match well ( https://github.com/streetcomplete/StreetComplete/blob/7995e29a2743c1bcd5754d7d1bae1c9b38a72389/app/src/main/java/de/westnordost/streetcomplete/osm/Things.kt#L12 ) though I am not fully sure so I wanted to ask for comments, if anyone has them.

See also https://github.com/streetcomplete/StreetComplete/blob/7995e29a2743c1bcd5754d7d1bae1c9b38a72389/app/src/main/java/de/westnordost/streetcomplete/osm/Place.kt

(it is also possible that it should go into neither one)

**Comment by mnalis** [comment]
It would be useful to add if missing, yes, as  _"These nodes are typically stand-alone"_ as stated in the wiki (so does not have issues like `barrier=gate` etc which must be placed on the ways they block). Probably Things overlay is better choice than Places.

But it should be accompanied with a quest that asks for `ref=*` on `aeroway=gate`, or it would not really be useful just by itself.

**Reply by matkoniecz** [reply]
I though also about using it for orientation - I was on AMS airport when returning from SOTM, and GPS was really spotty inside building. And gates would be excellent orientation points when mapping other features.

Though asking about `ref=` (especially if some quest can be extended to cover this) would be nice...

**Reply by HolgerJeromin** [reply]
Yes, in the layer the other thing or places are shown.
So its probably more useful to show gates while adding shops instead of things (which are more rare at the gate).

**Reply by matkoniecz** [reply]
I though about this but it is more a place. Though maybe it be shown for orientation like other shops are shown in POI quests?

**Reply by HolgerJeromin** [reply]
Places shows shops and restaurants, doesnt it?
But yes, gates combined with shops seems to be a good combination.

**Comment by westnordost** [comment]
Given that airport gates are rather points at which the travellers assemble, I'd say it should go into the same category as:

- bus station platforms
- taxi stands
- train station platforms
- `emergency=assembly_point`, `emergency=access_point`, `emergency=disaster_help_point`
- helipads
- waiting areas (if that can be tagged?)
- outdoor seating

**Reply by kmpoppe** [reply]
Is that a new public transport + safety overlay you speak of?

**Reply by westnordost** [reply]
no
